src/pages/login_page/login_page.scss

```diff
@@ -14,10 +14,10 @@
 }
 
 .form {
-  padding: 20px;
+  padding: 40px;
   flex-grow: 1;
-  display: inline;
+  display: inline;;
   text-align: center;
-  input[type="email"] { min-width: 200px; }
-  input[type="password"] { min-width: 200px; }
+  input[type="email"] {color: #1C1f2A; min-width: 300px; min-height: 25px; background-color: #b9dfeb; border-radius: 5px;}
```

Comment:
Add color definitions to https://github.com/onelogin/onelogin-demo-lab/blob/redesign/src/styles/colors.scss

and give them descriptive names like $onelogin-blue so its easier to read the CSS after a few months.

Reply:
done :-) $onelogin-blue and $onelogin-black + I have add $onelogin-gray